python/tests/test_adapt.py

```diff
@@ -40,16 +41,20 @@
 
 # 4 tags for the boundary + 1 tag for the interior
 if len(unique(tags.array())) != 7:
-    raise RuntimeError('Test failed: number of tags of generated mesh is different from 5')
+    raise RuntimeError('Test failed: number of tags of generated mesh is different from 7')
 
-if abs(new_mesh.num_vertices() - 833) > 80:
+if abs(new_mesh.num_vertices() - 905) > 80:
```

Comment:
why has this changed ?

Reply:
I guess pragmatic_adapt changed and the adapted mesh now has 905 vertices rather than 833